holoviews/core/util.py

```diff
@@ -246,7 +246,14 @@ def shortened_character_name(self_or_cls, c, eliminations=[], substitutions={},
         (as a list of tokens) by applying the eliminations,
         substitutions and transforms.
         """
-        name = unicodedata.name(c).lower()
+        ccmap = {'\a':unicodedata.name(u'a')}
+        if unicodedata.category(c) == 'Cc': # Handle control codes
```

Comment:
See [here](http://stackoverflow.com/questions/4324790/removing-control-characters-from-a-string-in-python). Seems 'Cc' is correct and makes sense to me if it stands for 'control code'.